gist/gist.py

```diff
@@ -166,11 +154,11 @@ def send(self, request, stem=None):
         if stem is not None:
             request.url = request.url + "/" + stem.lstrip("/")
 
-        session = requests.Session()
-        session.proxies = self.proxies
-        prepped = session.prepare_request(request)
+        prepped = self.session.prepare_request(request)
+        settings = self.session.merge_environment_settings(prepped.url, {}, 
+                                                           None, None, None)
```

Comment:
For some reason the proxy arg has to be a dict; it doesn't accept None like the rest of the options. Changed to use argument names for clarity.